.github/workflows/multi-approvers.js

```diff
@@ -2,49 +2,120 @@ const APPROVED = 'APPROVED';
 const COMMENTED = 'COMMENTED';
```

Comment:
Is there an option to use typescript here, or does that require some complex compilation step? /cc @verbanicm 

https://nodejs.org/en/learn/typescript/run-natively

Reply:
This looks like it is experimental and requires a special flag and I dont think we can control this for actions currently. So we would need to do the build process to "compile" into JS from TS.